client/src/components/LibraryNavigator/LibraryDataProvider.ts

```diff
@@ -17,24 +16,16 @@ import LibraryModel from "./LibraryModel";
 import { LibraryItem, LibraryType, TableType } from "./types";
 
 class LibraryDataProvider implements TreeDataProvider<LibraryItem> {
-  private _onDidChangeTreeData: EventEmitter<LibraryItem | undefined>;
-  private model: LibraryModel;
-  public dropMimeTypes: string[];
-  public dragMimeTypes: string[];
-  private extensionUri: Uri;
-
-  onDidChangeFile: Event<FileChangeEvent[]>;
+  private _onDidChangeTreeData = new EventEmitter<LibraryItem | undefined>();
 
   get onDidChangeTreeData(): Event<LibraryItem> {
     return this._onDidChangeTreeData.event;
   }
 
-  constructor(model: LibraryModel, extensionUri: Uri) {
-    this._onDidChangeTreeData = new EventEmitter<LibraryItem | undefined>();
-    this.model = model;
-    this.dragMimeTypes = ["application/vnd.code.tree.libraryDragAndDrop"];
```

Comment:
Is there going to be an issue with removing `dragMimeTypes` here? Did you test dragging a table into a sas program?

Reply:
I searched the whole project and this property is not used anywhere. It's indeed misleading. So I thought it's a good idea to clean this up.
I tried dragging a table into sas program and it does NOT work even on main branch. It's not impacted by this line.
I've gone ahead made an additional fix for the dragging issue. Please review. Thanks.